### DISCUSSION TITLE
blog/toast-notifications-in-aspnet-core/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#9] [General]
# blog/toast-notifications-in-aspnet-core/

In this article, we will learn about a simple package that can improve the user experience by adding Toast Notifications in ASP.NET Core MVC / Razor Applications. We must have all seen those beautiful Javascript toast notifications and have always tried to integrate it with our ASP.NET Application.

https://codewithmukesh.com/blog/toast-notifications-in-aspnet-core/

**Comment by Mwaseemzakir** [comment]
That is awesome work my friend, keep it coming :)
